db/postgres.go

```diff
@@ -30,6 +30,12 @@ const (
 //go:embed postgres
 var sql embed.FS
 
+type ExtraIndexes struct {
```

Comment:
Isso são informações de um _único_ índice, não de vários:

```suggestion
type ExtraIndex struct {
```